src/runtime_src/core/edge/drm/zocl/sched_exec.c

```diff
@@ -911,6 +915,65 @@ configure(struct sched_cmd *cmd)
 	return 0;
 }
 
+/**
+ * Gather ERT stats in ctrl command packet
+ * [1  ]      : header
+ * [1  ]      : custat version
+ * [1  ]      : ert version
+ * [1  ]      : number of cq slots
+ * [1  ]      : number of cus
+ * [#numcus]  : cu execution stats (number of executions)
+ * [#numcus]  : cu status (1: running, 0: idle)
+ * [#slots]   : command queue slot status
+ */
+static void
+cu_stat(struct sched_cmd *cmd)
+{
+	struct drm_zocl_dev *zdev = cmd->ddev->dev_private;
+	struct zocl_ert_dev *ert = zdev->ert;
+	struct sched_exec_core *exec = zdev->exec;
+	struct ert_packet *pkg;
+	int slot_idx;
+	int pkt_idx = 0;
+	int max_idx = (slot_size(cmd->ddev) >> 2) - 1;
+	int i;
+
+	SCHED_DEBUG("-> %s cq_slot_idx %d\n", __func__, cmd->cq_slot_idx);
+	slot_idx = cmd->cq_slot_idx;
+	pkg = cmd->packet;
+
+	/* custat version, update when changing layout of packet */
+	pkg->data[pkt_idx++] = 0x51a10000;
```

Comment:
Just curious, where is the 0x51a10000 from?

Reply:
https://github.com/Xilinx/XRT/blob/master/src/runtime_src/ert/scheduler/scheduler.cpp#L879